pkgs/development/libraries/audio/rtaudio/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, autoconf, automake, libtool, libjack2,  alsaLib, rtmidi }:
+
+stdenv.mkDerivation rec {
+  version = "4.1.2";
+  name = "rtaudio-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/thestk/rtaudio/archive/${version}.tar.gz";
+    sha256 = "08vyncrqvy5faiy8dffql2iqqyyj75hi5kkk5cb8axn5qlp2czqq";
+  };
+
+  buildInputs = [ autoconf automake libtool libjack2 alsaLib rtmidi ];
+
+  preConfigure = ''
+    ./autogen.sh --no-configure
+    ./configure
+  '';
+
+  meta = {
+    description = "A set of C++ classes that provide a common API for realtime audio input/output across Linux, Macintosh OS X , and Windows operating systems.";
+    homepage =  http://www.music.mcgill.ca/~gary/rtaudio/;
+    license = stdenv.lib.licenses.mit;
+    maintainers = [ stdenv.lib.maintainers.magnetophon ];
+    platforms = stdenv.lib.platforms.linux;
```

Comment:
I don't know this SW either, but it seems from the description I copy-pasted, that the cross platform part is important to them.